pkg/client/command_line.go

```diff
@@ -1,11 +1,12 @@
 package client
 
 import (
+	"fmt"
 	"os"
 	"path/filepath"
 
-	"github.com/mitchellh/go-homedir"
-	log "github.com/sirupsen/logrus"
+	yaml "gopkg.in/yaml.v2"
```

Comment:
Why is this in its own group?

Reply:
If you're on goland then adding a file watcher that runs gofmt and go goimports on file save is awesome.

Reply:
Just a mistake on my part; fixed